### PR TITLE
Add a Constable to Hurnscald

### PR DESCRIPTION
**Pull request by nashley**

### CODE REVIEW

**Comment by mekolat** [comment]
[23:55] Prsm: how about "the boss"
[23:58] Prsm: or call it "judge"
[23:58] Prsm: just judge
[00:00] Prsm: any name would be good
[00:01] Prsm: constable bob

**Comment by wushin** [comment]
@nashley move the NPC to the other side of walk way otherwise it would look funny if a GM opens the event portal.

**Comment by mekolat** [comment]
I would move it to where the debug is and move the debug (we don't care where the debug is since it's disabled on live server)

**Comment by wushin** [comment]
@nashley Yar, move constable to where debug NPC is and move debug NPC to where 4144 normally sits.

File: world/map/npc/009-1/constable.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+//
+
+009-1,52,33,0|script|Constable Bob|419
+{
+    mes "[Constable Hank Atkins]";
+    if (strcharinfo(0)=="Prsm")
```
Comment (mekolat) [inline world/map/npc/009-1/constable.txt]: @nashley don't add a special handler for a specific player. make it say the same thing to everyone

**Pull request merged**